### PR TITLE
Add URL_CONFIG

### PR DESCRIPTION
**Pull request by fgourichon**
@ouranos for review

### CODE REVIEW

File: src/app/index.route.coffee [hunk 1]
```diff
@@ -39,16 +39,22 @@ angular.module 'mnoEnterpriseAngular'
...
+
             if I18N_CONFIG.enabled
-              $window.location.href = "/#{$translate.use()}#{URI.login}"
+              if URL_CONFIG.after_sign_out_url
+                $window.location.href = "#{logout_url}"
+              else
+                $window.location.href = "/#{$translate.use()}#{URI.login}"
             else
-              $window.location.href = "#{URI.login}"
+              $window.location.href = "#{logout_url}"
```
Comment (ouranos) [inline src/app/index.route.coffee]: Any reason why we had it before?

File: src/app/index.route.coffee [hunk 2]
```diff
@@ -39,16 +39,22 @@ angular.module 'mnoEnterpriseAngular'
...
             AnalyticsSvc.logOut()
+
+            logout_url = URL_CONFIG.after_sign_out_url || "#{URI.login}"
+
             if I18N_CONFIG.enabled
-              $window.location.href = "/#{$translate.use()}#{URI.login}"
+              if URL_CONFIG.after_sign_out_url
+                $window.location.href = "#{logout_url}"
+              else
+                $window.location.href = "/#{$translate.use()}#{URI.login}"
```
Comment (ouranos) [inline src/app/index.route.coffee]: It's the same as the previous code.

Comment (alexnoox) [inline src/app/index.route.coffee]: `URI.login` has a `/` as first character

Comment (alexnoox) [inline src/app/index.route.coffee]: all good ;)

File: src/app/index.route.coffee (hunk 1 above)
Comment (alexnoox) [inline src/app/index.route.coffee]: No, but I suppose it could be removed.

**Comment by fgourichon** [comment]
Removed interpolations

**Pull request merged**